hr_holidays_legal_leave/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+HR Holidays Legal Leave
+=======================
+
+Allows the define which holiday type is to be considered legal/annual leave type.
+Currently Odoo assumes by default leave type with limit=False. This is a problem if
```

Comment:
@dufresnedavid I think @tarzan0820 was trying to say that Odoo is raising an erro when you try to set the remaining days of employee, if it doesn't find at leats one holiday status with limit = False, see https://github.com/odoo/odoo/blob/8.0/addons/hr_holidays/hr_holidays.py#L513, but for this i really don't know of they are coutnries where you can take legal leaves in advance from next year...probably this is the case that @tarzan0820 want to represent from this statement.